governance/polls/SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 11, 2024.md

```diff
@@ -0,0 +1,51 @@
+---
+title: SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 11, 2024 
+summary: Signal your support or opposition to increasing the supply cap maximum on SparkLend Ethereum to 6,000 WBTC.
+discussion_link: https://forum.makerdao.com/t/mar-6-2024-proposed-changes-to-sparklend-for-upcoming-spell/23791
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-03-11T16:00:00
+end_date: 2024-03-14T16:00:00
+---
+# Poll: SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 11, 2024  
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of SparkLend. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, March 11 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
+* Increase the WBTC Supply Cap `max` by 1,000 WBTC, from 5,000 to **6,000 WBTC**.
```

Comment:
nit - consider adding a definition for the `max`

Reply:
In future polls, we'll include some definitions or documentation.